### ISSUE TITLE
lxqt-meta: SLOT vs MY_PV in dependencies

### ISSUE THREAD

**Issue by ff2000** [dracwyrm/lxqt#2]
I wondered why emerge didn't show me version 0.13 for installing.
```
ebuild  /var/repositories/lxqt/lxqt-base/lxqt-meta/lxqt-meta-0.13.0.ebuild manifest
Error(s) in metadata for 'lxqt-base/lxqt-meta-0.13.0':
  RDEPEND: Invalid atom (lxqt-base/lxqt-about:0.13*), token 14
```
After changing all `CAT/PKG:${MY_PV}` into `=CAT/PKG-${MY_PV}` `ebuild lxqt-meta-0.13.0 manifest` succeeded and I can install v0.13.0 now. I don't know if some packages would need to be ":${SLOT}" instead, but this fix is consistent with the non-USE-dependencies.

Fingers crossed that it now compiles and runs fine :)

**Comment by Dessa** [comment]
oh wow i wonder why this issue isn't closed: i specified wrong number in the commit :| this should be fixed now as of #3

**Comment by ff2000** [comment]
Because this is "issue #2" and you said "fixes #1" ;)
Will close this manually.

Thx for the fix!

**Comment by dracwyrm** [comment]
@Dessa I completely missed that it was the wrong number as well. Thanks for everyone reporting and helping fix it. :) My time is very limited sometimes.
